projboard/tests/test_client.py

```diff
@@ -113,3 +114,60 @@ def test_aboutpage(client):
     template_names = set(tmpl.origin.template_name for tmpl in response.templates)
     # And check if that 'about/about.html' is in the set.
     assert 'about/about.html' in template_names
+
+
+@pytest.fixture
+@pytest.mark.django_db
+def User2(client):
+    return User.get_user_by_nickname("User2")
+
+
+@pytest.mark.django_db
+def test_read_new_article(client, world_cup_article, User2):
```

Comment:
The name does not explain what the test need todo test, 
Maybe short description

def test_name()
"""
Test description
"""

Reply:
done